.github/workflows/build_docs.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Install Dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install "napari/[all]"
+          python -m pip install "napari/[pyside,dev]"
```

Comment:
wonder if we should just stick with pyqt and even consider moving circleCI to pyqt.
pyside (and pyside6 especially) has been the more flaky backend...